docusaurus/docs-internals/editor-controls/unit-controls/range.md

```diff
@@ -0,0 +1,272 @@
+| `devices?`           | `"all"` \| `"desktop"` \| `"responsive"`                                                                                                                                                   | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices |
+| `disabled?`          | `boolean`                                                                                                                                                                                  | `false` | Configure the condition under which the control is disabled or enabled                                                                                                                                                           |
+| `helper?.content`    | `string`                                                                                                                                                                                   |    -    | If provided, displays an icon next to the label. When hovering over this icon, a tooltip with additional information appears                                                                                                     |
+| `helper?.position`   | `"top-start"` \| `"top"` \| `"top-end"` \| `"right-start"` \| `"right"` \| `"right-end"` \| `"bottom-end"` \| `"bottom"` \| `"bottom-start"` \| `"left-end"` \| `"left"` \| `"left-start"` | `"top"` | Specifies the position of the tooltip relative to the helper icon                                                                                                                                                                |
+| `config.min?`        | `number`                                                                                                                                                                                   |    -    | The minimum value allowed on the range scale                                                                                                                                                                                     |
+| `config.max?`        | `number`                                                                                                                                                                                   |    -    | The maximum value allowed on the range scale                                                                                                                                                                                     |
```

Comment:
`min` and `max` also has a default

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/182c9acb-9044-463a-a5c2-0b0caae615ee)